applications/TopologyOptimizationApplication/python_scripts/topology_optimizer_factory.py

```diff
@@ -0,0 +1,370 @@
+# ==============================================================================
+#  KratosTopologyOptimizationApplication
+#
+#  License:         BSD License
+#                   license: TopologyOptimizationApplication/license.txt
+#
+#  Main authors:    Philipp Hofer, https://github.com/PhiHo-eng
+#                   Erich Wehrle, https://github.com/e-dub
+#  based on original file from
+#                   Baumgärtner Daniel, https://github.com/dbaumgaertner
+#                   Octaviano Malfavón Farías
+#                   Eric Gonzales
+#
+# ==============================================================================
+
+# ------------------------------------------------------------------------------
+# Imports
+# ------------------------------------------------------------------------------
+
+# Making KratosMultiphysics backward compatible with python 2.6 and 2.7
+from __future__ import print_function, absolute_import, division
+
+# importing the Kratos Library
+from KratosMultiphysics import *
+from KratosMultiphysics.TopologyOptimizationApplication import *
```

Comment:
```suggestion
import KratosMultiphysics as KM
import KratosMultiphysics.TopologyOptimizationApplication as KratosTopOpt
```

using wildcard imports is basically never a good idea and should be generally avoided (not Kratos specific, applies to all python)